source/material_model/rheology/friction_options.cc

```diff
@@ -0,0 +1,849 @@
+                           "Pore fluid pressure can be taken into account by specifying the 'Effective friction "
+                           "factor', which uses $\\mu* = \\mu\\big(1-\\frac{P_f}{\\sigma_n} \\big)$. "
+                           "Reasonable values for a and b are 0.01 and 0.015, respectively, see \\cite{sobolev_modeling_2017}. "
+                           "In ASPECT the state variable is confined to positive values: if it becomes $<=0$ during the computation "
+                           "it is set to 1e-10. The friction angle is set to 0 if negative. "
+                           "\n\n"
+                           "\\item ``rate and state dependent friction plus linear slip weakening'': ToDo: all, this is an empty model atm. "
+                           "Method taken from \\cite{sobolev_modeling_2017}. The friction coefficient is computed as "
+                           "$\\mu = \\mu_{st} + a \\cdot ln\\big( \\frac{V}{V_{st}} \\big) + b \\cdot ln\\big( \\frac{\\Theta V_{st}}{L} \\big) - \\Delta \\mu(D)$"
+                           "where D is the slip at point in fault at the first timestep of earthquake. "
+                           "\n\n"
+                           "\\item ``slip rate dependent rate and state dependent friction'': The rate and state "
+                           "parametera and the critical slip distance L are made slip rate dependent. The friction "
+                           "coefficient is computed as in 'rate and state dependent friction'. But a and L are not "
+                           "constant, but are computed as follws, see \\citep{Im_im_slip-rate-dependent_2020} for details."
+                           "$a(V) = a_0 + s_a log_{10}\\left(\\frac{V_a+V}{V_a}\\right)$ and "
+                           "$L(V) = L_0 + s_L log_{10}\\left(\\frac{V_L+V}{V_L}\\right)$."
+                           "So a and L have a log linear dependence on velocity with slopes od $s_a$ and $s_L$. "
+                           "Parameters in their paper have the following values: "
+                           "$L_0=10\\mu m$, $s_L=60\\mu m$, $V_L=100\\mu m/s$ and "
+                           "$a_0=0.005$, $s_a=0.0003$, $V_a=100\\mu m/s$. "
+                           "In ASPECT the initial values $a_0$ and $L_0$ are the rate and state friction parameters "
+                           "indicated in 'Critical slip distance' and 'Rate and state parameter a function'."
+                           "\n\n"
+                           "\\item ``regularized rate and state friction'': The friction coefficient is computed using: "
+                           "$\\mu = a\\cdot sinh^{-1}\\left[\\frac{V}{2V_0}exp\\left(\\frac{\\mu_0+b\cdot ln(V_0\theta/L)}{a}\\right)\\right]$ "
+                           "This is a high velocity approxiamtion and regularized version of the classic rate and state friction. "
```

Comment:
```suggestion
                           "This is a high velocity approximation and regularized version of the classic rate and state friction. "
```
Most IDEs have spelling control as well, does yours?